app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -1,5 +1,12 @@
 <% title t('titles.enter_2fa_code.one_time_code') %>
 
+<%= render AlertComponent.new(type: :info, class: 'margin-bottom-4 usa-alert--info-time') do %>
+  <%= t(
+        'instructions.mfa.otp.time_remaining',
+        countdown: CountdownComponent.new(expiration: @presenter.otp_expiration).render_in(self),
+      ).html_safe %>
```

Comment:
I could make the string `time_remaining_html` instead, but I'm not sure how I feel about `_html` locale strings which themselves don't contain the HTML. For example, it'd be nice to lint `_html` strings to expect they include HTML, which would be hard to do with dynamically-injected HTML. Maybe I'm overthinking it 🤷

Reply:
> Linter doesn't like `html_safe` so I'll probably just switch to `_html` suffix instead.

Updated in 308f1c8af.

Reply:
Sounds like a good idea to me. I updated to `_html` suffix for the interpolated bit in cc1bd81.